R/string2factor.R

```diff
@@ -29,20 +29,20 @@
 #'
 #' @examples
 #' library(modeldata)
-#' data(okc)
+#' data(Sacramento)
 #'
-#' rec <- recipe(~ diet + location, data = okc)
+#' rec <- recipe(~ city + zip, data = Sacramento)
 #'
 #' make_factor <- rec %>%
-#'   step_string2factor(diet)
+#'   step_string2factor(city)
```

Comment:
The Sacremento data set isn't good for this step since city and zip are already factor variables.